tests/src/test/java/org/eclipse/hono/tests/http/HttpTestBase.java

```diff
@@ -896,6 +895,7 @@ public void testUploadMessagesWithTtdThatReplyWithCommand(final TestContext ctx)
                 });
     }
 
+
```

Comment:
there is no real change to this file, is there? If so, I think it should be removed from this commit.